### PR TITLE
scheduler: Store mesos-start-time - first known starting/running state

### PR DESCRIPTION
**Pull request by mforsyth**

### CODE REVIEW

File: scheduler/test/cook/test/mesos/scheduler.clj [hunk 1]
```diff
@@ -836,6 +836,27 @@
...
+      (let [job-id (create-dummy-job conn
+                                     :user "mforsyth"
+                                     :job-state :job.state/running
+                                     :retry-count 3)
+            task-id "task-mesos-start-time"
+            instance-id-a (create-dummy-instance conn job-id
+                                                 :instance-status :instance.status/unknown
+                                                 :task-id task-id
+                                                 :reason :unknown)
+            mesos-start-time (fn [] (-> conn d/db
```
Comment (wyegelwel) [inline scheduler/test/cook/test/mesos/scheduler.clj]: nitpick: put `d/db` on a separate line

Comment (mforsyth) [inline scheduler/test/cook/test/mesos/scheduler.clj]: 👍

File: scheduler/test/cook/test/mesos/scheduler.clj [hunk 2]
```diff
@@ -836,6 +836,28 @@
...
+                                        d/db
+                                        (d/entity [:instance/task-id task-id])
+                                        :instance/mesos-start-time))]
+        (is (nil? (mesos-start-time)))
+        (async/<!! (sched/handle-status-update conn driver fenzo
+                                               (make-dummy-status-update task-id :unknown :task-staging)))
+        (is (nil? (mesos-start-time)))
+        (async/<!! (sched/handle-status-update conn driver fenzo
+                                               (make-dummy-status-update task-id :unknown :task-running)))
+        (is (not (nil? (.getTime (mesos-start-time)))))))
```
Comment (dposada) [inline scheduler/test/cook/test/mesos/scheduler.clj]: Maybe send another status update after this and assert that the persisted time doesn't change?

Comment (mforsyth) [inline scheduler/test/cook/test/mesos/scheduler.clj]: @dposada good idea.  👍

**Pull request merged**